spec/controllers/rating/applications_controller_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 
-describe Rating::ApplicationsController do
+describe Rating::ApplicationsController, type: :controller do
```

Comment:
I thought we don't need to set the type because we have:
https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/32e7940730b453245cbdfd42a7ec9f1e625b0771/spec/support/database_cleaner.rb#L26

Reply:
For having `database_cleaner` work as before, yes. But maybe someone wants to add a specific config for controller-types specs, e.g. including certain matchers there...?

I actually did not put much thought into this 😉 

However, I think it would be good to have, especially if we finally start to add feature-specs (we should def. do this) - so that we can distinguish which parts of the suite require a headless browser, etc.
Plus, we could split the build to run feature-specs and non feature specs in parallel...

Reply:
If we are to add the `type` key, can we also change the `describe` to `RSpec.describe`? No, no need to update this PR, just thinking about it while reading this.